### ISSUE TITLE
Requires nightly/pre-release rust

### ISSUE THREAD

**Issue by ChrisJAllan** [yshui/index_camera_passthrough#1]
With a standard rust install, building gives the error
```
error[E0554]: `#![feature]` may not be used on the stable release channel
 --> openvr_sys/src/lib.rs:1:1
  |
1 | #![feature(untagged_unions)]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^

For more information about this error, try `rustc --explain E0554`.
```

The readme should note the requirement of pre-release rust.

**Comment by yshui** [comment]
Thanks for pointing this out!

**Comment by yshui** [comment]
Fixed in 76a2bb1
